src/Features/Core/Diagnostics/Log/DiagnosticLogAggregator.cs

```diff
@@ -61,26 +63,30 @@ public AnalyzerInfo(DiagnosticAnalyzer analyzer, AnalyzerActions analyzerActions
                 CLRType = analyzer.GetType();
                 Telemetry = telemetry;
 
-                Counts[0] = analyzerActions.CodeBlockEndActionsCount;
-                Counts[1] = analyzerActions.CodeBlockStartActionsCount;
-                Counts[2] = analyzerActions.CompilationEndActionsCount;
-                Counts[3] = analyzerActions.CompilationStartActionsCount;
-                Counts[4] = analyzerActions.SemanticModelActionsCount;
-                Counts[5] = analyzerActions.SymbolActionsCount;
-                Counts[6] = analyzerActions.SyntaxNodeActionsCount;
-                Counts[7] = analyzerActions.SyntaxTreeActionsCount;
+                Counts[0] = analyzerActions.CodeBlockActionsCount;
```

Comment:
It should be fine.

From: Srivatsn Narayanan [mailto:notifications@github.com]
Sent: Thursday, March 5, 2015 10:53 AM
To: dotnet/roslyn
Cc: HeeJae Chang
Subject: Re: [roslyn] Diagnostics API changes for registering compilation actions, code block actions, and end actions (#1043)


In src/Features/Core/Diagnostics/Log/DiagnosticLogAggregator.cs<https://github.com/dotnet/roslyn/pull/1043#discussion_r25890193>:

> @@ -61,26 +63,30 @@ public AnalyzerInfo(DiagnosticAnalyzer analyzer, AnalyzerActions analyzerActions

>                  CLRType = analyzer.GetType();

>                  Telemetry = telemetry;

>

> -                Counts[0] = analyzerActions.CodeBlockEndActionsCount;

> -                Counts[1] = analyzerActions.CodeBlockStartActionsCount;

> -                Counts[2] = analyzerActions.CompilationEndActionsCount;

> -                Counts[3] = analyzerActions.CompilationStartActionsCount;

> -                Counts[4] = analyzerActions.SemanticModelActionsCount;

> -                Counts[5] = analyzerActions.SymbolActionsCount;

> -                Counts[6] = analyzerActions.SyntaxNodeActionsCount;

> -                Counts[7] = analyzerActions.SyntaxTreeActionsCount;

> +                Counts[0] = analyzerActions.CodeBlockActionsCount;

Would changing the order of these actions interfere with telemetry that we've already collected? If so, we should probably just put the new actions at the end. @heejaechang<https://github.com/heejaechang> would know.

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/1043/files#r25890193>.